samples/snippets/speech_to_storage_beta.py

```diff
@@ -14,12 +14,21 @@
 
 
 # [START speech_transcribe_with_speech_to_storage_beta]
+# google.cloud.speech_v1.types.RecognizeResponse
```

Comment:
This type should be SpeechRecognitionResult as per https://cloud.google.com/speech-to-text/docs/async-recognize#upload_your_transcription_results_to_a_bucket

Reply:
```RecognizeResponse``` seems to be compatible with the string representation of the returning object. I am constrained by what’s returned by ```results_string = blob.download_as_string()```. ```SpeechRecognitionResult``` does not include formatting that allows ```results```, which is what the conversion to ```.proto``` is processing.

Reply:
Oh, that documentation is not correct then. Now looking at it I see it should have been LongRunningRecognizeResponse.  RecognizeResponse has subset of fields of LongRunningRecognizeResponse. Essentially they are the same, but more correct one to use is LongRunningRecognizeResponse.

Reply:
Updated to ```LongRunningRecognizeResponse```.